srcpkgs/libgphoto2/template

```diff
@@ -14,7 +14,7 @@ license="LGPL-2.1-or-later"
 homepage="http://www.gphoto.org"
 changelog="https://raw.githubusercontent.com/gphoto/libgphoto2/master/NEWS"
 distfiles="${SOURCEFORGE_SITE}/gphoto/${pkgname}-${version}.tar.bz2"
```

Comment:
```suggestion
distfiles="https://github.com/gphoto/libgphoto2/releases/download/v${version}/libgphoto2-${version}.tar.bz2"
```
seems they moved to github a while back. seems the files have the same checksum, so should be easy to switch

Reply:
Add that change, should I prioritize github when available from other hosts ?

Reply:
not necessarily. in this case the upstream was clear about moving things to github